doc/content/reference/configuration/join-server.md

```diff
@@ -3,3 +3,7 @@ title: "Join Server Options"
 description: ""
 weight: 6
 ---
+
+## General configuration
```

Comment:
I'm pretty sure we use Title Case for titles

Reply:
For consistency with the other files in this section, can we use "options" instead of "configuration"?